compiler-core/src/warning.rs

```diff
@@ -196,6 +196,23 @@ a future version of Gleam."
                 )),
             },
             Self::Type { path, warning, src } => match warning {
+                type_::Warning::UnusedFunctionBody { location } => Diagnostic {
+                    title: "Unused function body.".into(),
```

Comment:
To be consistent with #2506 this shouldn't have a full stop at the end of the title